app/src/main/res/layout/child_comment_item.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    >
+  <RelativeLayout
+      android:id="@+id/outer_layout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/comment_gray"
+      android:paddingBottom="14dp"
+      android:paddingEnd="15dp"
+      android:paddingLeft="30dp"
+      android:paddingRight="15dp"
+      android:paddingStart="30dp"
+      android:paddingTop="14dp"
+      >
+
+    <ImageView
+        android:id="@+id/user_icon"
+        android:layout_width="34dp"
+        android:layout_height="34dp"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:src="@drawable/icon_user"
+        tools:scaleX="0.7"
+        tools:scaleY="0.7"
+        />
+
+    <TextView
+        android:id="@+id/user_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="10.3dp"
+        android:layout_marginStart="10.3dp"
+        android:layout_marginTop="6dp"
+        android:layout_toEndOf="@id/user_icon"
+        android:layout_toRightOf="@id/user_icon"
+        android:ellipsize="end"
+        android:maxLines="1"
+        android:textSize="@dimen/text_size_extra_small"
+        android:textStyle="bold"
+        tools:text="Mary Pollaki"
+        />
+
+    <TextView
+        android:id="@+id/added_date_pos2"
```

Comment:
why 2 on added_date_pos2

Reply:
now i remember... I copy+pasted the whole comment_layout.xml so I wouldn't touch the view. I will open a ticket to refactor this layout and names!

Reply:
fun fact: the layout is from 05/08/2016 !